meinberlin/react/contrib/ControlBarFilterPills.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import django from 'django'
+import { Pill } from './Pill'
+
+const translated = { filterRemove: django.gettext('Remove filter %s') }
+
+const getFilterRemoveText = (label) => {
+  return django.interpolate(translated.filterRemove, [label])
+}
+
+export const ControlBarFilterPills = ({ filters: _filters, onRemove }) => {
+  const filters = _filters.filter(f => f.value && f.value !== '')
+  return filters.length
+    ? (
```

Comment:
more of a personal choice but I find this rather hard to read, wouldn't something like
```
if filters.length == 0 {
  return null
}
return (....)
```
be a bit easier?